sql/catalyst/src/main/scala/org/apache/spark/sql/types/StructType.scala

```diff
@@ -284,7 +286,8 @@ case class StructType(fields: Array[StructField]) extends DataType with Seq[Stru
     val nonExistFields = names -- fieldNamesSet
     if (nonExistFields.nonEmpty) {
       throw new IllegalArgumentException(
-        s"Field ${nonExistFields.mkString(",")} does not exist.")
+        s"""Fields ${nonExistFields.mkString(", ")} does not exist.
```

Comment:
nit: `nonExistFields` can contain only one field, so maybe `Fields` -> `Field(s)`?

Reply:
"Field(s) ... do(es) not exist." like this?

Reply:
`Not existing field(s): ${nonExistFields.mkString(", ")}` ?

Reply:
done